curriculum/challenges/english/25-front-end-development/lab-newspaper-layout/6735a7370e0ae93a4577c689.md

```diff
@@ -0,0 +1,423 @@
+                diseases, it's important to eat a balanced diet that includes a variety of nutrient-rich foods. Here are
+                some tips for eating a balanced diet:</p>
+        </article>
+
+        <article class="small-article3">
+            <h3>Travel: Top 10 Destinations for 2025</h3>
+            <p>Traveling is one of the best ways to experience new cultures, meet new people, and see the world. If
+                you're looking for inspiration for your next trip, here are the top 10 destinations for 2025:</p>
+        </article>
+
+        <article class="secondary-article">
+            <h2>Technology: The Rise of AI</h2>
+            <p>Artificial intelligence (AI) is changing the way we live and work. From self-driving cars to virtual
+                assistants, AI is revolutionizing the world around us. But what exactly is AI, and how does it work? In
+                this article, we'll explore the rise of AI and its impact on society.</p>
+        </article>
+
+        <figure class="cover-image">
+            <img src="https://cdn.freecodecamp.org/curriculum/labs/volcano.jpg" alt="Cover Image">
+        </figure>
+    </main>
+</body>
+
+</html>
+```
+
+```css
+body {
+    font-family: 'Georgia', 'Times New Roman', Times, serif;
+    padding: 0;
+    margin: 0;
+    background-color: #f0f0f0;
+}
+
+.newspaper-layout {
+    display: grid;
+    grid-template-areas:
+        "title title title"
+        "feature-article feature-article cover-image"
+        "secondary-article secondary-article cover-image"
+        "small-article1 small-article2 small-article3";
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto 1fr 1fr 1fr;
+    gap: 15px;
+    background-color: #fff;
+    padding: 15px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    box-sizing: border-box;
+    text-align: justify;
+}
+
+.title {
+    grid-area: title;
+    background-color: #1B1B32;
+    color: white;
+    padding: 15px;
+    text-align: center;
+}
+
+.title h1 {
+    font-size: 2.5rem;
+    line-height: 1.3;
+    margin: 0;
+}
+
+.feature-article,
+.secondary-article {
+    background-color: #f9f9f9;
+    padding: 15px;
+    font-size: 1rem;
+}
+
+.feature-article {
+    grid-area: feature-article;
+}
+
+.secondary-article {
+    grid-area: secondary-article;
+}
+
+.small-article1,
+.small-article2,
+.small-article3 {
+    background-color: #e9e9e9;
+    padding: 15px;
+    font-size: 0.9rem;
+}
+
+.small-article1 {
+    grid-area: small-article1;
+}
+
+.small-article2 {
+    grid-area: small-article2;
+}
+
+.small-article3 {
+    grid-area: small-article3;
+}
+
+.cover-image {
+    grid-area: cover-image;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #ddd;
+    padding: 15px;
+    margin: 0;
+}
+
+.cover-image img {
+    max-width: 100%;
+    height: 350px;
+    width: 350px;
+    border-radius: 5px;
+}
```

Comment:
with not requiring any style for the image it takes a lot of space and breaks the layout